src/connections/destinations/catalog/actions-fullstory-cloud/index.md

```diff
@@ -26,19 +26,21 @@ The FullStory cloud mode destination sends information about your users and rela
 
 ### Benefits of FullStory Cloud Mode (Actions)
 
-- Works with FullStory’s latest data capture APIs
+- Enrich autocapture data with FullStory's latest data capture APIs
```

Comment:
I'm noticing the diff on this file is highlighting a single quote change here and single space changes elsewhere. Noting just so we can verify we're moving toward intended char codes instead of these being unexpected char code changes.

Reply:
i think these might be unexpected char code changes, i copied the contents from shortcut and it changed a lot of the whitespace.

Reply:
looks like the whitespace is my IDE, i think these are normal but i fixed the commas to use these slanted ones: ’